community/tools/graphql2chartjs/example/hasura/public-schema.sql

```diff
@@ -0,0 +1,197 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 11.2 (Ubuntu 11.2-1.pgdg16.04+1)
+-- Dumped by pg_dump version 11.2
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+SET row_security = off;
+
+--
+-- Name: public; Type: SCHEMA; Schema: -; Owner: rxrngbplmkiody
+--
+
+CREATE SCHEMA public;
+
+
+ALTER SCHEMA public OWNER TO rxrngbplmkiody;
```

Comment:
There is owner info all over this file.

Why is this file required of we have migrations already?